docs/migration/3_0.md

```diff
@@ -179,6 +179,7 @@ There can be more than one cart entry with the same product code. So now they ar
 - `AuthConfig` no longer extends `OccConfig`.
 - `login` and `revoke` endpoints were removed from `OccConfig`. `login` endpoint is now available under `tokenEndpoint` property in `AuthConfig`. `revoke` endpoint is available under `revokeEndpoint` property in `AuthConfig`.
 - `storageSync` configuration for `auth` branch in ngrx store was removed. State of token, userId is now synchronized with `AuthStatePersistenceService`. Override this service if you want to sync more properties to localStorage (eg. `refresh_token`).
+- `anonymousConsentsStoreConfigFactory` was removed. Function has been replaced by `anonymousConsentsStatePersistenceFactory`. It requires param type `AnonymousConsentsStatePersistenceService` and it doesn't return `StateConfig`, only function allows for initialization the synchronization between state and browser storage.
```

Comment:
These details are not important. More important is mention of storageSync config. Eg.
```suggestion
- `storageSync` configuration for `anonymous-consents` branch in ngrx store was removed. State is now synchronized with `AnonymousConsentsStatePersistenceService`. Override this service if you want to sync more/less properties to localStorage.
```